middleware/queue/source/manager/handle.cpp

```diff
@@ -592,7 +593,10 @@ namespace casual
 
             auto spawn = []( auto& entity)
             {
-               entity.process.pid = common::process::spawn( state::entity::path( entity), {});
+               entity.process.pid = common::process::spawn(
+                  state::entity::path( entity),
+                  {},
+                  { instance::variable( instance::Information{ entity.configuration.alias})});
```

Comment:
Hmm, we might wan't to _extend_ `process::spawn` to make it more strict what is "needed", instead "communicating" with environment variables... 

Either way, I don't think this commit/feature should improve that. Just a thought.